lib/mindee/client.rb

```diff
@@ -128,6 +136,8 @@ def parse_queued(
     #   Doesn't need to be set in the case of OTS APIs.
     # @param all_words [Boolean] Whether to extract all the words on each page.
     #   This performs a full OCR operation on the server and will increase response time.
+    # @param full_text [Boolean] Whether to include the full OCR text response in compatible APIs.
+    #  This performs a full OCR operation on the server and will increase response time.
```

Comment:
why does the user need to specify this for `parsed_queued`? This should only be set when sending the request, not when retrieving it.